docs/remote/containers.md

```diff
@@ -278,27 +278,36 @@ When you rebuild and reopen in your container, the features you selected will be
 
 You'll get IntelliSense when editing the `"features"` property in the `devcontainer.json` directly:
 
-![Intellisense when modifying terraform feature](images/containers/features-intellisense.png)
+![Intellisense when modifying terraform Feature](images/containers/features-intellisense.png)
 
-Built-in features are sourced from the [script library](https://github.com/microsoft/vscode-dev-containers/tree/main/script-library/docs) folder in the vscode-dev-containers repo, but the Remote - Containers extension and GitHub Codespaces include an **early preview** for creating your own dev container features. For example, you can reference these as follows:
+The **Remote-Containers: Configure Container Features** command allows you to update an existing configuration.
+
+The Features sourced in VS Code UI come from the [`devcontainers/features` repository](https://github.com/devcontainers/features).
+
+### Folder structure
+
+A feature is a self contained entity in a folder with at least a `devcontainer-feature.json` and `install.sh` entrypoint script:
 
-```json
-"features": {
-    "your-github-id-or-org/your-repository/feature-name@v0.0.1": "latest"
-}
+```
++-- feature
+|    +-- devcontainer-feature.json
+|    +-- install.sh
+|    +-- (other files)
 ```
 
-The form and format of these custom features is still in flux, but you can try creating your own dev container feature using the [dev-container-features-template](https://github.com/microsoft/dev-container-features-template) sample repository.  Let us know what you think!
+### Features proposal and distribution
 
-The **Remote-Containers: Configure Container Features** command allows you to update an existing configuration.
+Features are an active proposal in the [open dev container specification](https://containers.dev). You can review the [Features proposal](https://github.com/devcontainers/spec/issues/61) and [greater information about how Features work](https://containers.dev/implementors/features/).
+
+You can try creating your own dev container feature using the [feature-template](https://github.com/devcontainers/feature-template) sample repository. There is also [more information about their distribution](https://containers.dev/implementors/features-distribution/) in the spec.
```

Comment:
If this moved up above the proposal line, you could narrow down to just the features and distribution proposals.